samples/drivers/adc/README.rst

```diff
@@ -32,7 +24,15 @@ sure that the ADC is enabled (``status = "okay";``).
 In addition to that, this sample requires an ADC channel specified in the
 ``io-channels`` property of the ``zephyr,user`` node. This is usually done with
 a devicetree overlay. The example overlay in the ``boards`` subdirectory for
-the :ref:`nucleo_l073rz_board` can be easily adjusted for other boards.
+the ``nucleo_l073rz`` board can be easily adjusted for other boards.
+
+The ADC resolution to be used can be specified in the ``adc-resolution``
+property of the ``zephyr,user`` node. Also channel configuration (settings
+like gain, reference, or acquisition time) can be specified in devicetree,
+in ADC controller child nodes. See :zephyr_file:`boards/nrf52840dk_nrf52840.overlay
+<samples/drivers/adc/boards/nrf52840dk_nrf52840.overlay>` for an example of
+such configuration. If these parameters are not specified in devicetree,
+default values, supposed to be supported by most ADCs, are used for them.
```

Comment:
Could we have a `zephyr,resolution` dts property for each channel?

Reply:
Yes, good suggestion. I'll add it and `zephyr,oversampling` as well.

Reply:
Added.